pandas/io/json/_normalize.py

```diff
@@ -531,7 +532,14 @@ def _recursive_extract(data, path, seen_meta, level=0):
             raise ValueError(
                 f"Conflicting metadata name {k}, need distinguishing prefix "
             )
-        result[k] = np.array(v, dtype=object).repeat(lengths)
+        if v and is_list_like(v[0]):
```

Comment:
As an alternative can we not just do something like:

```python
arr = np.empty(1, dtype=object)
arr[0] = v
arr = arr.repeat(lengths)
```

Less than ideal but I think still gets us to the same place? The branching now is a little tough to follow

Reply:
@mroeschke suggested not to construct the np array, what I wrote before is
```python
result[k] = np.array(v).repeat(lengths, axis=1).tolist()
```

Reply:
I think @WillAyd's suggestion is good if it works. My main issue before was one path calling `tolist()` and the other returning an `np.ndarray`

Reply:
Late response. Unfortunately this won't work. When v is a `list`, `arr` will be `[list(...)]`, which raises boardcasting error. Classified discussions may still be necessary. (Now both paths return `list` for consistency and auto-type-infering.) @mroeschke